src/hooks/useLoadUpdateDealDeps.ts

```diff
@@ -102,14 +102,8 @@ const useLoadUpdateDealDeps: UseLoadUpdateDealDeps = (dealId) => {
         getContactsService,
         {
             select: (data) => {
-                return (get(data, ["results"], []) as Contact[]).map(({
-                    id: contactId,
-                    properties: {
-                        firstname: firstName,
-                        lastname: lastName
-                    },
-                }) => {
-                    return getOption(contactId, getFullName({ firstName, lastName }));
+                return (get(data, ["results"], []) as Contact[]).map((c) => {
```

Comment:
We shouldn't need to typecast here if the types are accurate. Maybe they're been lost to `lodash` inwhich case a native alternative would be better, `data?.results ?? []`

Reply:
fixed